config/jobs/kubernetes/sig-cluster-lifecycle/manifests.yaml

```diff
@@ -18,7 +18,7 @@ periodics:
     testgrid-dashboards: sig-cluster-lifecycle-all
     testgrid-tab-name: periodic-manifest-lists
     testgrid-alert-email: kubernetes-sig-cluster-lifecycle+testgrid@googlegroups.com
-    description: "OWNER: sig-cluster-lifecycle; an e2e test for verifying manifest list images at k8s.gcr.io"
+    description: "OWNER: sig-cluster-lifecycle; an e2e test for verifying manifest list images at registry.k8s.io"
```

Comment:
this is the only line that matches the existing pull requests, it's a small enough change that it seemed better to just include it.

the other changes in this PR are different and based on closely inspecting each remaining reference.